introduction-to-git/src/person/DimitarApostolov.java

```diff
@@ -2,7 +2,7 @@
 
 public class DimitarApostolov extends AbstractPerson{

```

Comment:
You are missing space before curly bracket, please add it or use eclipse formatting.